pages/news/2020-02-20-mixxx-accepted-for-gsoc-2020.html

```diff
@@ -0,0 +1,24 @@
+title: Mixxx accepted for GSoC 2020!
+author: Jan Holthuis
+date: 2020-02-20 20:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Mixxx has been [accepted as a mentoring organization](https://summerofcode.withgoogle.com/organizations/6577142898360320/) for [Google Summer of Code 2020](https://summerofcode.withgoogle.com/), a global program that aims to bring more student developers into the world of open source development.
+
+This means that students have to possiblity to work on Mixxx full-time as a 3 month programming project and receive a stipend sponsored by Google. For more information, check out the [Google Summer of Code FAQ](https://developers.google.com/open-source/gsoc/faq#students).
+
+We have prepared a list of [GSoC 2020 Project Ideas for Mixxx](https://mixxx.org/wiki/doku.php/gsoc2020ideas) and added it to our wiki, but we're always interesed in new ideas. Taking a look at our [GSoC advice page](https://mixxx.org/wiki/doku.php/gsocadvice) is recommended if you're planning to apply. The best way to get started is to participate in the Mixxx community, have a look at our [GitHub repository](https://github.com/mixxxdj/mixxx) and possibly making a small code contribution to show that you're able to familiarize yourself with our codebase.
```

Comment:
typo: interested

Reply:
The best way to get started is to participate in the Mixxx community, **so** have

Reply:
"and possibly making a small code contribution" -> "and consider making a small code contribution"